src/olympia/accounts/views.py

```diff
@@ -211,15 +217,18 @@ def inner(self, request):
 
 def add_api_token_to_response(response, user, set_cookie=True):
     # Generate API token and add it to the json response.
-    payload = jwt_api_settings.JWT_PAYLOAD_HANDLER(user)
-    token = jwt_api_settings.JWT_ENCODE_HANDLER(payload)
+    data = {
+        'secret': user.get_session_auth_hash(),
```

Comment:
This looks like a session ID, not a secret. It also looks like this will be part of the publicly accessible base64 string, right? Does it have to be called secret?